client/reader/tags-page/trending-tags.tsx

```diff
@@ -0,0 +1,54 @@
+import { TagResult } from './controller';
+
+interface Props {
+	trendingTags: TagResult[];
+}
+
+export default function TrendingTags( { trendingTags }: Props ) {
+	if ( ! trendingTags ) {
+		return null;
+	}
+
+	// put the data into a format that can easily be rendered as a two column table.
+	const tagRows: TagResult[][] = [];
+	for ( let i = 0; i < trendingTags.length; i += 2 ) {
+		const tagOne = trendingTags[ i ];
+		const tagTwo = trendingTags[ i + 1 ];
+		tagRows.push( [ tagOne, tagTwo ] );
+	}
+
+	const toTitleCase = ( text: string ) => {
```

Comment:
Micro-optimization: it might be a good idea to move the generic function definitions out of this function, so they would be defined only once.